### PR TITLE
neon db branching

### PR DESCRIPTION
**Pull request by Bilalm354**

### CODE REVIEW

File: db/client.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { drizzle } from "drizzle-orm/vercel-postgres";
-import { sql } from "@vercel/postgres";
+import { neon } from "@neondatabase/serverless";
+import { drizzle } from "drizzle-orm/neon-http";
 
-// Use this object to send drizzle queries to your DB
+const sql = neon(process.env.DATABASE_URL!);
```
Comment (haffi96) [inline db/client.ts:4]: whats the difference?

Comment (Bilalm354) [reply db/client.ts:4]: supabase might still be a better option but if this branching works that'll be cool

Comment (haffi96) [reply db/client.ts:4]: nice, supabase has branching too

Comment (haffi96) [reply db/client.ts:4]: but neon will probably have better vercel integration

**Pull request merged**